libavcodec/jpeg2000.h

```diff
@@ -37,24 +37,26 @@
 
 enum Jpeg2000Markers {
     JPEG2000_SOC = 0xff4f, // start of codestream
+    JPEG2000_CAP = 0xff50, // Fixed information marker segments
     JPEG2000_SIZ = 0xff51, // image and tile size
-    JPEG2000_COD,          // coding style default
-    JPEG2000_COC,          // coding style component
+    JPEG2000_COD = 0xff52, // coding style default
```

Comment:
This will need to be split into a separate commit:

* one commit to add `JPEG2000_CTSY_HTJ2K_F` and `JPEG2000_CTSY_HTJ2K_M` detection
* one commit to improve the marker enumeration and detection of CAP and CPF marker
* one commit to fix code style issues (if any)